my-cypress-project/cypress/pages/realWordAppPage.js

```diff
@@ -0,0 +1,117 @@
+export default class LoginPage {
+    constructor() {
+        this.userInfo = {
+            firstName: "Olena",
+            lastName: "Gofman",
+            username: "OlenaGofman",
+            password: "secret",
+            confirmPassword: "secret"
+        }
+
+        this.date = new Date().getTime();
+    }
+    openRealWorldApp() {
+        cy.visit('http://localhost:3000/')
+
+    }
+    signUp() {
+        cy.get('[data-test="signup"]').click()
+        cy.get('#firstName').type(this.userInfo.firstName)
+        cy.get('#lastName').type(this.userInfo.lastName)
+        cy.get('#username').type(this.userInfo.username + this.date)
+        cy.get('#password').type(this.userInfo.password, { log: false })
+        cy.get('#confirmPassword').type(this.userInfo.confirmPassword, { log: false })
+        cy.get('.MuiButton-label').click()
```

Comment:
If in the test there is not assertion on the page switching to the next one - it is recommended to add an assertion in this test (page switched to signed-up page).